infrastructure/stack/security_groups.tf

```diff
@@ -0,0 +1,37 @@
+data "aws_vpc" "vpc" {
+  id = var.vpc_id
+}
+
+resource "aws_security_group" "vpc_ingress" {
+  name        = "${var.environment}-content-api-vpc-ingress"
+  description = "Allow traffic between services and the rest of the VPC"
```

Comment:
Is this description actually accurate? I know it's copied, but let's not copy round incorrect information.

Reply:
Ha, I had this thought when I copied it - I don't think so either!